less/paperhive/login.less

```diff
@@ -0,0 +1,82 @@
+.ph-login-icon {
+  width: 18px;
+  height: 18px;
+  vertical-align: baseline;
+}
+
+.ph-login-separator {
+  color: #E0E0E0;
+}
+
+.ph-login-line-separator {
+  width: 1px;
+  background-color: #E0E0E0;
+  height: 12em;
+}
+
+.ph-flex-container {
+  display: -webkit-flex;
+  display: flex;
+  -webkit-justify-content: center;
+  justify-content: center;
+}
+
+.ph-flex-item {
+  -webkit-align-self: center;
+  align-self: center;
+  -webkit-flex: 3;
+  flex: 3;
+}
+
+.ph-flex-item-1 {
+  -webkit-align-self: center;
+  align-self: center;
+  -webkit-flex: 1;
+  flex: 1;
+}
+
+.ph-inner-flex-container {
+  display: -webkit-flex;
+  display: flex;
+  -webkit-flex-direction: column;
+  flex-direction: column;
+}
+
+.ph-inner-flex-item {
+  -webkit-align-self: center;
+  align-self: center;
+  -webkit-flex: 1;
+  flex: 1;
```

Comment:
In firefox the vertical rule is not displayed and the spacing is wrong:

![screenshot from 2016-02-10 23-18-26](https://cloud.githubusercontent.com/assets/1874116/12963737/9db36ce8-d04c-11e5-82ec-2c095d85396a.png)

Removing this `flex: 1` fixes the issue (and it still works in chrome). I'm no flexbox expert so I'm not aware of the consequences. @jengeb: do we need this? :)